sanic/server.py

```diff
@@ -121,9 +122,20 @@ def on_message_complete(self):
     # -------------------------------------------- #
 
     def write_response(self, response):
+        """Attempts to write the resposne to the transport
+
+        This will attempt to smartly handle any type of python object with
```

Comment:
Might sound a little better if you say something like ...`attempt to intelligently handle`...